src/sinks/pulsar.rs

```diff
@@ -47,9 +47,11 @@ enum BuildError {
 pub struct PulsarSinkConfig {
     /// The endpoint to which the Pulsar client should connect to.
     #[serde(alias = "address")]
+    #[configurable(metadata(docs::examples = "pulsar://127.0.0.1:6650"))]
     endpoint: String,
```

Comment:
Are there specific formats of the string which are acceptable, or required?
For example, is a port required? could a hostname work in addition to an IP address?
Is "pulsar://" scheme required in the string?
In other components, we have outlined the constraints a bit. 
Including different formats in additional examples can also help.

Reply:
Let me look into this.

Reply:
I added docs about this.